.github/workflows/e2e_libvirt.yaml

```diff
@@ -100,11 +100,6 @@ jobs:
           [ -f ~/.ssh/id_rsa ] || \
             ssh-keygen -t rsa -f ~/.ssh/id_rsa -N ""
 
-          # Newest version of kcli does not index old Ubuntu images like 20.04
```

Comment:
Btw the newest kcli does index 20.04 again thanks to Fabiano ;-) (not that it'd be important for this PR)